lua/hbac/command/init.lua

```diff
@@ -0,0 +1,54 @@
+local subcommands = require("hbac.command.subcommands")
+
+local notify_opts = { title = "Hbac" }
+
+local M = {
+	subcommands = {},
+}
+
+M.subcommands.close_unpinned = function()
+	subcommands.close_unpinned()
+	vim.notify("Closed unpinned buffers", "info", notify_opts)
+end
+
+M.subcommands.toggle_pin = function()
+	local bufnr, pinned_state = subcommands.toggle_pin()
+	vim.notify(bufnr .. " " .. pinned_state, "info", notify_opts)
+end
+
+M.subcommands.toggle_all = function()
```

Comment:
I see a potential problem with `toggle_all` in that it, and correct me if I'm wrong, it doesn't update with the pin status of buffers. Lets say we `toggle_all` to pin all buffers with 1, 2, 3 open and then close a buffer and open some more: 1, 2, 4, 5. What should be the behavior when calling `toggle_all` again? Judging by the current code we're going to unpin all buffers, which might not be ideal.

Reply:
That's a good point. Maybe the simplest way to deal with this is to have 2 separate commands. Otherwise the logic gets kinda bizzare. With 2 commands the user can expect the same thing every time pin all pins all buffers. Unpin all unpins all. Simple to implement and simple for the user to use.

Reply:
Sounds good, I think I had just gotten used to calling the command twice quickly with a convenient sequence but I realize that's kinda hacky. Will split.